src/Engine/Graphics/RenderEntities.h

```diff
@@ -78,6 +78,10 @@ struct RenderVertexSoft {
     float u = 0;
     float v = 0;
     float flt_2C = 0;
+
+    bool operator==(const RenderVertexSoft& other) const {
+        return vWorldPosition == other.vWorldPosition;
+    }
```

Comment:
This `operator==` looks sus to me, is this intended?

Reply:
Yeah most definitely whack. Will move this comparison to the code instead.